framework/tests/returnsreceiverlombok/BuilderMethodRef.java

```diff
@@ -0,0 +1,31 @@
+import java.util.Optional;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+import org.checkerframework.common.returnsreceiver.qual.*;
+
+@Builder
+@Accessors(fluent = true)
+public class BuilderMethodRef {
+    @Getter @Setter @lombok.NonNull String foo;
+    @Getter @Setter Object bar;
+
+    public static void test(Optional<Object> opt) {
+        BuilderMethodRefBuilder b = builder().foo("Hello");
+        // :: error: (methodref.receiver.bound.invalid)
```

Comment:
This is not a valid error.  We should either use `@SuppressWarnings` or add a comment indicating this is not a valid error, pointing to #2931